partitura/score.py

```diff
@@ -2540,8 +2600,8 @@ def create_variant_part(self):
 
         return part
 
-
-def iter_unfolded_parts(part):
+# DEPRECATED
+def DEPRECATEDiter_unfolded_parts(part):
```

Comment:
I'd completely delete them before merging, as they are replaced with equivalent functions of the same name and functionality, but with different internals. But renaming + warning/decorator is fine with me too.

Reply:
ok, then I think it would be better to completely delete them before merging, since they could cause some confusion and will stop working altogether with newer versions.